components/scream/src/physics/p3/p3_functions_f90.cpp

```diff
@@ -1314,40 +1363,34 @@ void update_prognostic_liquid_f(Real qc2qr_accret_tend_, Real nc_accret_tend_, R
   *nr_    = t_h(5);
 }
 
-void ice_deposition_sublimation_f(Real qi_incld_, Real ni_incld_, Real T_atm_, Real qv_sat_l_,
-				  Real qv_sat_i_, Real epsi_, Real abi_, Real qv_,
-				  Real* qv2qi_vapdep_tend_, Real* qi2qv_sublim_tend_, Real* ni_sublim_tend_, Real* qc2qi_berg_tend_)
+void ice_deposition_sublimation_f(Real qi_incld, Real ni_incld, Real t_atm, Real qv_sat_l, Real qv_sat_i, Real epsi, Real abi, Real qv, Real inv_dt, Real* qidep, Real* qi2qv_sublim_tend, Real* ni_sublim_tend, Real* qiberg)
 {
-  using P3F = Functions<Real, DefaultDevice>;
+#if 0
```

Comment:
This entire region is disabled by `#if 0`

Reply:
@jgfouca - 
1. it's odd these `#if 0` comments are here - I suspect they must have been added by gen-boiler because I certainly didn't add these lines...
2. I don't think the newest versions of our tests use <func>_f any more. I meant to ask you whether I can actually just delete <func>_f references...

Reply:
@PeterCaldwell , you're right. Since we now have a full cxx mode for P3, there's no need for f90 p3 routines to call their cxx counterparts. I will issue a PR to clean this up

Reply:
Ok, I'll delete the 2 _f functions you noticed are `#if 0` in this PR, but leave the rest of them for you to deal with later.